trading_report/src/main.ts

```diff
@@ -114,14 +138,19 @@ async function main() {
   appendFile('trade_log.txt', `${SYMBOL} Trading at ${(new Date()).toISOString()}`, () => {});
   appendFile('trade_log.txt', `
 Openbook trades
-  Bought ${opbTotalBuysAmount} for ${opbTotalBuysValue / (opbTotalBuysAmount + .0000000001)}, net notional: ${opbTotalBuysValue}
-  Sold ${opbTotalSellsAmount} for ${opbTotalSellsValue / (opbTotalSellsAmount + .0000000001)}, net notional: ${opbTotalSellsValue}
+  Bought ${opbTotalBuysAmount} for ${opbBuyAvgPrice}, net notional: ${opbTotalBuysValue}
+  Sold ${opbTotalSellsAmount} for ${opbSellAvgPrice}, net notional: ${opbTotalSellsValue}
 `, () => {}
 );
   appendFile('trade_log.txt', `
 Jupiter trades
-  Bought ${jupTotalBuysAmount} for ${jupTotalBuysValue / (jupTotalBuysAmount + .0000000001)}, net notional: ${jupTotalBuysValue}
-  Sold ${jupTotalSellsAmount} for ${jupTotalSellsValue / (jupTotalSellsAmount + .0000000001)}, net notional: ${jupTotalSellsValue}
+  Bought ${jupTotalBuysAmount} for ${jupBuyAvgPrice}, net notional: ${jupTotalBuysValue}
+  Sold ${jupTotalSellsAmount} for ${jupSellAvgPrice}, net notional: ${jupTotalSellsValue}
+`, () => {}
+);
+appendFile('trade_log.txt', `
```

Comment:
Is this going to break the parsing in the UI

Reply:
I don't see trade_log.txt being parsed anywhere. Think it just displays the most recent blob of text? Could be missing something though?